src/main/java/Mariogatchi/Item.java

```diff
@@ -0,0 +1,67 @@
+package Mariogatchi;
+
+import java.io.Serializable;
+
+public abstract class Item implements Serializable {
+
+    // set of Environments that the mariogatchi.Item can be used in
+    private Set<Environment> eligibleAreas = new Set<Environment>;
+
+    // a mapping of Effects to their corresponding values
+    private Map<Effect, Integer> effectValues = new Map<Effect, Integer>;
```

Comment:
This can probably be implemented as String to Integer.

Reply:
I made Effect an enum so is now a String constant.